pontoon/base/static/js/translate.js

```diff
@@ -1853,13 +1853,13 @@ var Pontoon = (function (my) {
         // Tab: Select suggestions
         if (!$('.menu').is(':visible') && key === 9 && !e.ctrlKey) {
 
-          // Prevent in complex FTL mode
+          // Rich FTL editor: ignore tab key
           if (self.fluent.isFTLEditorEnabled() && self.fluent.isComplexFTL()) {
             return;
           }
 
-          // Special case in FTL source editor
-          if ($('#translation').is(':visible') || $('#ftl').is('.active')) {
+          // Source FTL editor: insert tab character
+          if ($('#ftl:visible').is('.active')) {
```

Comment:
isn't that `self.fluent.isComplexFTL()`?

Reply:
No, FTLEditor is the form-based AKA rich FTL editor as seen here:
https://pontoon.mozilla.org/sl/firefox/browser/browser/preferences/main.ftl/?string=174655